samples/videoDecodePerf/CMakeLists.txt

```diff
@@ -92,13 +92,16 @@ if(HIP_FOUND AND FFMPEG_FOUND AND ROCDECODE_FOUND AND Threads_FOUND AND rocprofi
     else()
       target_compile_definitions(${PROJECT_NAME} PUBLIC USE_AVCODEC_GREATER_THAN_58_134=1)
     endif()
+    target_compile_definitions(${PROJECT_NAME} PUBLIC USE_FFMPEG=1)
 else()
     message("-- ERROR!: ${PROJECT_NAME} excluded! please install all the dependencies and try again!")
     if (NOT HIP_FOUND)
         message(FATAL_ERROR "-- ERROR!: HIP Not Found! - please install ROCm and HIP!")
     endif()
     if (NOT FFMPEG_FOUND)
-        message(FATAL_ERROR "-- ERROR!: FFMPEG Not Found! - please install FFMPEG!")
+        # FFMPEG compile flag
+        target_compile_definitions(${PROJECT_NAME} PUBLIC USE_FFMPEG=0)
```

Comment:
Defining `USE_FFMPEG=0` in this context is incorrect. If FFMPEG is not available, the application will not be built at all (refer to line 68 of this CMakeLists.txt). Therefore, please refrain from defining the USE_FFMPEG flag for this application.